frankenphp.go

```diff
@@ -334,14 +338,12 @@ func Init(options ...Option) error {
 
 // Shutdown stops the workers and the PHP runtime.
 func Shutdown() {
+	stopWatcher()
 	stopWorkers()
 	close(done)
 	shutdownWG.Wait()
```

Comment:
I think this can be removed now since it is awaited in `stopWorkers()`

Reply:
This is needed since the watcher sessions have to be destroyed.

Reply:
I will rename the method to 'drainWatcher'